src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/LaunchProfilesDebugLaunchProvider.cs

```diff
@@ -262,9 +262,9 @@ internal static VsDebugTargetInfo4 GetDebuggerStruct4(IDebugLaunchSettings info)
             {
                 var processStartupInfo = new VsDebugStartupInfo
                 {
-                    hStdInput = unchecked((uint)info.StandardInputHandle.ToInt32()),
-                    hStdOutput = unchecked((uint)info.StandardOutputHandle.ToInt32()),
-                    hStdError = unchecked((uint)info.StandardErrorHandle.ToInt32()),
+                    hStdInput = unchecked((IntPtr)(uint)info.StandardInputHandle.ToInt32()),
```

Comment:
Can we go straight from `info.StandardInputHandle` to `IntPtr` without going though Int32 and uint first?

That is, are we able to simply these expressions somehow?

Reply:
Yes, in fact `info.StandardInputHandle` is an `IntPtr` so none of these are needed.